src/models/encounter/usePatchEncounter.js

```diff
@@ -0,0 +1,84 @@
+import { useState } from 'react';
+import axios from 'axios';
+import { get } from 'lodash-es';
+import { formatError } from '../../utils/formatters';
+
+export default function usePatchEncounter() {
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState(null);
+  const [success, setSuccess] = useState(false);
+
+  const updateProperties = async (encounterId, dictionary) => {
+    const dictionaryCopy = { ...dictionary };
+    if ('gps' in dictionaryCopy) {
+      dictionaryCopy.decimalLatitude = get(
+        dictionaryCopy,
+        ['gps', 0],
+        null,
+      );
+      dictionaryCopy.decimalLongitude = get(
+        dictionaryCopy,
+        ['gps', 1],
+        null,
+      );
+      delete dictionaryCopy.gps;
+    }
+
+    // delete dictionaryCopy.taxonomy;
```

Comment:
Why commented out? Just a bookmark for the future?

Reply:
i should remove this actually..